underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -78,27 +79,30 @@
    * Constructs a new instance of {@link GCSUnderFileSystem}.
    *
    * @param uri the {@link AlluxioURI} for this UFS
+   * @param ufsConf the configuration for this ufs
```

Comment:
nit: use consistent capitalization of UFS (the line above uses "UFS", you use "ufs")